trackma/ui/qt/models.py

```diff
@@ -346,14 +342,14 @@ def data(self, index, role):
         elif role == QtCore.Qt.DecorationRole:
             return self.thumbs.get(row)
         elif role == QtCore.Qt.BackgroundRole:
-            t = self.results[row].get('type')
-            if t == utils.TYPE_TV:
+            t = self.results[row].get('type').name
+            if t == 'TV':
                 return QtGui.QColor(202, 253, 150)
-            elif t == utils.TYPE_MOVIE:
+            elif t == 'Movie':
                 return QtGui.QColor(150, 202, 253)
-            elif t == utils.TYPE_OVA:
+            elif t == 'OVA':
                 return QtGui.QColor(253, 253, 150)
-            elif t == utils.TYPE_SP:
+            elif t == 'Special':
```

Comment:
I'm still unhappy with this change and it was als the primary reason for my earlier comment.

To be precise, there should be an enum for these 4 literals, likely inside `utils`. Then the site integrations need to provide mappings to the globally known types, so that the interfaces can have a single source of knowledge they can rely on. Incidentally, this is closer to what it was before this PR.

I while explaining this in more detail, I quickly figured out that reimplementing it how I see it should be would be much quicker. In the process I also replaced all the other constants with enums.

Will open a PR soon.